website/examples/date-picker-dialog.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { DayPicker } from 'react-day-picker';
+import { usePopper } from 'react-popper';
+
+import { format, isValid, parse } from 'date-fns';
+import FocusTrap from 'focus-trap-react';
+
+export default function Example() {
+  const [selected, setSelected] = React.useState<Date>();
+  const [inputValue, setInputValue] = React.useState<string>('');
+  const [isOpen, isOpenSet] = React.useState(false);
```

Comment:
I usually call the setter this way:

```suggestion
  const [isPopperOpen, setIsPopperOpen] = React.useState(false);
```